### ISSUE TITLE
[徵才] Rida.ai -  Fullstack JavaScript developer / Mobile Engineer (Flutter)

### ISSUE THREAD

**Issue by leilaKao** [f2etw/jobs#935]
# 公司名稱
Rida.ai 

# 公司網站
https://rida.ai

# 職缺與薪資範圍
## Fullstack JavaScript developer 
80~160K/Month
## Mobile Engineer 
70~140K/Month

# 職務說明
#### Fullstack JavaScript developer 
We are currently looking for a Fullstack JavaScript developer who will work closely with the team to build and maintain new features as well as enhance existing ones. As a developer at Rida, you need to be passionate not only about shipping code but also quality. Above all, we are looking for someone who cares deeply about the user experience.
But before we go on to talk about the role in more detail, here’s a peek into our engineering process and tech stack.

#### Mobile Engineer
We are currently looking for our first in-house Mobile Engineer (Flutter) who will help to create a cutting-edge mobile experience for our users. As a developer at Rida, you need to be passionate not only about shipping code but also quality. Above all, we are looking for someone who cares deeply about the user experience.
But before we go on to talk about the role in more detail, here’s a peek into our engineering process and tech stack.

## Engineering process:
 2-week sprints
• Weekly release cycle
• Scrum methodology
• Collaboration tools: Jira, Confluence
• Code reviews
• PRDs (prepared and signed off by the Product team before any development)

## Tech stack:
• Frontend: VueJS, Apollo
• Backend: Node.js
• Database: GQL, Postgres
• Cloud: AWS, GCP
• Deployment: CI/CD using CircleCI
• Microservices

#### Fullstack JavaScript developer 
• Design, develop, test, deploy, and integrate systems with JavaScript frameworks
• Consistently write effective code that is stable, secure, scalable, and efficient
• Accurately define effort requirements and manage personal workflow to ensure that deliverables are completed on time

#### Mobile Engineer
• Develop, test, deploy, and maintain production quality mobile apps using Flutter
• Consistently write and document code that is stable, secure, scalable, and efficient
• Manage individual project priorities, deadlines, and deliverables
• Participate in creating technical specifications and performing code reviews with other engineers

# 職缺要求
#### Fullstack JavaScript developer 
• 2+ years of relevant experience building production-level applications
• Experience working with projects built on Node.js
• Experience working with databases like PostgreSQL, MongoDB
• Familiarity with at least one of the cloud platforms (AWS/Azure/GCP) and their services
• Interested in software development process automation—testing automation, CI/CD, etc.
• Implementing solutions using best practices and technologies
• Working experience in a startup

#### Mobile Engineer
• 2+ years mobile dev experience
• At least one complex app published on app stores

## Nice to have experience
• GraphQL
• JavaScript
• Native or React Native
• Apple Human Interface and Google Material Design guidelines
• Automated testing
• Working in a startup

## Soft skills
• Eye for design
• Attention to detail
• Collaborative

# 公司介紹
Rida is a fast-growing Singapore startup that is transforming deliveries, one batch at a time.
Let’s face it - with rising e-commerce and multiple third-party logistics (3PL) riders headed, oftentimes, in the same direction, package, route, and vehicle optimizations have gone haywire. The result—high delivery costs for merchants and jams for the rest of us!
At Rida.ai, we are tackling logistics inefficiencies headlong.
Our algorithms batch orders and aggregate 3PL quotes, finding the best delivery options. Merchants make use of our wide choice of delivery windows to dispatch orders in a smarter manner. And platforms and marketplace use our APIs to launch delivery services in a matter of days!
In short, fewer rides but more deliveries in the ecosystem. And for merchants, higher revenues at lower costs.

# 聯絡方式
```
Send your CV to careers@rida.ai
```
